src/docs/patterns/overlay.mdx

```diff
@@ -71,25 +77,35 @@ The modal dialog appears in the center of the screen in front of the primary con
 
 Simple modals should have a title, a descriptive message, and action buttons for users to respond to the prompt or dismiss the dialog. These modals should be no larger than is necessary to contain their contents.
 
+<ImageGrid
```

Comment:
General grammar comment: punctuation goes inside the quotes when it's at the end of a sentence (in the images and in the captions).

Reply:
I think there might be some I18N concerns here. I will remove the quotation marks